examples/04_pong/main.rs

```diff
@@ -127,19 +133,19 @@ impl Processor<Arc<Mutex<Context>>> for PongProcessor {
                     // Store left plank dimensions for later use in ball processing
                     left_dimensions = plank.dimensions;
                     // If `W` is pressed and plank is in screen boundaries then move up
-                    if ctx.input_handler.key_down(VirtualKeyCode::W) {
+                    if input_handler.key_down(VirtualKeyCode::W) {
                         if plank.position + plank.dimensions[1]/2. < 1. {
                             plank.position += plank.velocity * delta_time;
                         }
                     }
                     // If `S` is pressed and plank is in screen boundaries then move down
-                    if ctx.input_handler.key_down(VirtualKeyCode::S) {
+                    if input_handler.key_down(VirtualKeyCode::S) {
                         if plank.position - plank.dimensions[1]/2. > -1. {
                             plank.position -= plank.velocity * delta_time;
                         }
                     }
                     // Set translation[0] of renderable corresponding to this plank
-                    local.translation[0] = left_boundary + plank.dimensions[0]/2.0
+                    local.set_translation_index(0 , left_boundary + plank.dimensions[0]/2.0);
```

Comment:
I actually got it to work through the use of DerefMut so you can use it exactly how you would modify the inner wrapped part. Eventually this will be changed to use a TrackedStorage instead so there is no deref abuse and you can use the components without getters and setters.